components/ephemeral_storage/browser/ephemeral_storage_browsertest.cc

```diff
@@ -0,0 +1,164 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/net/brave_network_delegate_browsertest.h"
```

Comment:
you definitely should not be doing this, just add these tests to brave_network_delegate_browsertest and do a scoped enabling of the feature

Reply:
If you prefer all of the tests for ephemeral storage to be in the `brave_network_delegate_browstertest.cc`, we can do this.

Reply:
We will still need to subclass `BraveNetworkDelegateBrowserTest` because 
```
[19561:19561:0911/111412.678737:FATAL:feature_list.cc(39)] Check failed: feature_overrides_allowed. FeatureList overrides must happen in the test constructor, before BrowserTestBase::SetUp() has run.
```

Reply:
In the end, we plan to add a decent number of tests for ephemeral storage, so we've maintained the `EphemeralStorageBrowserTest` fixture in its own file, but are no longer sublcassing `BraveNetworkDelegateTest`.